src/FffGcodeWriter.cpp

```diff
@@ -828,19 +828,47 @@ void FffGcodeWriter::addMeshLayerToGCode(SliceDataStorage& storage, SliceMeshSto
     }
 }
 
+static void parseIntegerList(const char *input, std::vector<int> *output) {
```

Comment:
It's desirable to just return the vector rather than require the caller to generate one.